doc/Programming-guide/Key-platform-features/Data-collection/Kafka-log-appender/index.md

```diff
@@ -0,0 +1,292 @@
+    "pluginTypeName":"Kafka",
+    "applicationId":"5",
+    "applicationToken":"82635305199158071549",
+    "name":"Sample Kafka log appender",
+    "description":"Sample Kafka log appender",
+    "headerStructure":[
+        "KEYHASH",
+        "VERSION",
+        "TIMESTAMP",
+        "TOKEN",
+        "LSVERSION"
+    ],
+    "maxLogSchemaVersion":2147483647,
+    "minLogSchemaVersion":1,
+    "tenantId":"1",
+    "jsonConfiguration":"{\"kafkaServers\":[{\"host\":\"localhost\",\"port\":9092}],\"topic\":\"kaa\",\"useDefaultPartitioner\":true,\"partitionCount\":1,\"kafkaKeyType\":{\"org.kaaproject.kaa.server.appenders.kafka.config.gen.KafkaKeyType\":\"NULL\"},\"executorThreadPoolSize\":1,\"bufferMemorySize\":33554432,\"kafkaCompression\":{\"org.kaaproject.kaa.server.appenders.kafka.config.gen.KafkaCompression\":\"NONE\"},\"kafkaAcknowledgement\":{\"org.kaaproject.kaa.server.appenders.kafka.config.gen.KafkaAcknowledgement\":\"ONE\"},\"retries\":0}"
+}
+```
+
+Example result:
+
+```json
+{
+    "applicationId": "5",
+    "applicationToken": "82635305199158071549",
+    "confirmDelivery": true,
+    "createdTime": 1466497790454,
+    "createdUsername": "devuser",
+    "description": "Sample Kafka log appender",
+    "headerStructure": [
+        "KEYHASH",
+        "VERSION",
+        "TIMESTAMP",
+        "TOKEN",
+        "LSVERSION"
+    ],
+    "id": "131077",
+    "jsonConfiguration": "{\"kafkaServers\":[{\"host\":\"localhost\",\"port\":9092}],\"topic\":\"kaa\",\"useDefaultPartitioner\":true,\"partitionCount\":1,\"kafkaKeyType\":{\"org.kaaproject.kaa.server.appenders.kafka.config.gen.KafkaKeyType\":\"NULL\"},\"executorThreadPoolSize\":1,\"bufferMemorySize\":33554432,\"kafkaCompression\":{\"org.kaaproject.kaa.server.appenders.kafka.config.gen.KafkaCompression\":\"NONE\"},\"kafkaAcknowledgement\":{\"org.kaaproject.kaa.server.appenders.kafka.config.gen.KafkaAcknowledgement\":\"ONE\"},\"retries\":0}",
+    "maxLogSchemaVersion": 2147483647,
+    "minLogSchemaVersion": 1,
+    "name": "Sample Kafka log appender",
+    "pluginClassName": "org.kaaproject.kaa.server.appenders.kafka.appender.KafkaLogAppender",
+    "pluginTypeName": "Kafka",
+    "tenantId": "1"
+}
+```
+
+# Playing with Kafka log appender
+
+To check out Kafka log appender you can play with [Data collection demo](https://github.com/kaaproject/sample-apps/tree/master/datacollectiondemo/source).
+Download [Kaa Sandbox](http://www.kaaproject.org/download-kaa/) then set up it and go to Data collection demo application.
+
+>**NOTE:**
+> Kafka must be installed, running and reachable from Kaa to complete this example. For details about Kafka installation refer to
+[official Apache documentation](https://kafka.apache.org/07/quickstart.html).
+> Kaa use Zookeeper service started on port 2181. And Before starting Kafka you need to change zookeeper port from 2181 to for example 2183.
+> In the Kafka installation directory edit next property files: <br/>
```

Comment:
Could this be re-written to something like below?

For this, you will need to edit properties files (they are located within the Kafka installation directory) like below: